bitsandbytes/diagnostics/main.py

```diff
@@ -49,19 +51,24 @@ def main():
 
     print_header("OTHER")
     cuda_specs = get_cuda_specs()
-    print("CUDA specs:", cuda_specs)
+    if HIP_ENVIRONMENT:
+        rocm_specs = f" rocm_version_string='{cuda_specs.cuda_version_string}',"
+        rocm_specs += f" rocm_version_tuple={cuda_specs.cuda_version_tuple}"
+        print(f"{BNB_BACKEND} specs:{rocm_specs}")
+    else:
+        print(f"{BNB_BACKEND} specs:{cuda_specs}")
```

Comment:
Yeah, this smells even more like `CudaSpecs` and `RocmSpecs` should be separate dataclasses.

Reply:
This requires updates to cextensions and additional testing. Could we use this workaround for now and make the change after alpha release?